prow/config/labels.yaml

```diff
@@ -235,6 +235,11 @@ default:
       description: Should cherry pick this PR to release-8.5 branch.
       target: prs
       addedBy: anyone
+    - name: needs-cherry-pick-tidb-cloud
```

Comment:
It is placed under https://github.com/lilin90/configs/blob/tidb-cloud-label/prow/config/labels.yaml#L80. Is it expected?